firebase-perf/src/main/java/com/google/firebase/perf/metrics/AppStartTrace.java

```diff
@@ -129,29 +128,17 @@ public static void setLauncherActivityOnResumeTime(String activity) {
     // no-op, for backward compatibility with old version plugin.
   }
 
-  public static AppStartTrace getInstance() {
-    return instance != null ? instance : getInstance(TransportManager.getInstance(), new Clock());
-  }
-
-  static AppStartTrace getInstance(TransportManager transportManager, Clock clock) {
-    if (instance == null) {
-      synchronized (AppStartTrace.class) {
-        if (instance == null) {
-          instance =
-              new AppStartTrace(
-                  transportManager,
-                  clock,
-                  ConfigResolver.getInstance(),
-                  new ThreadPoolExecutor(
-                      CORE_POOL_SIZE,
-                      MAX_POOL_SIZE,
-                      /* keepAliveTime= */ MAX_LATENCY_BEFORE_UI_INIT + 10,
-                      TimeUnit.SECONDS,
-                      new LinkedBlockingQueue<>()));
-        }
-      }
-    }
-    return instance;
+  public AppStartTrace() {
```

Comment:
I am not sure if "De-singletoning this hides AppStartTrace.getInstance() from customers. " Customers can still call new AppStartTrace() I think.